pkg/proxy/util/utils_test.go

```diff
@@ -383,55 +383,55 @@ func TestMapCIDRsByIPFamily(t *testing.T) {
 			ipString:        []string{"fd00:20::1/64"},
 			wantIPv6:        false,
 			expectCorrect:   nil,
-			expectIncorrect: []string{"fd00:20::1/64"},
+			expectIncorrect: []string{"fd00:20::/64"},
```

Comment:
Ugh, maybe fix all these (and the ones in `traffic_test.go`) as a separate commit before this one? (The fixes are correct even before the change to `NewDetectLocalByCIDR`; we should not have been testing it using bad CIDRs.)

Reply:
Collected in the first commit in this PR, and I also made a new PR https://github.com/kubernetes/kubernetes/pull/122655, if we want better history tracking. But please lgtm that PR fast if we shall use it.